app/components/maps/festival-nav/festival-nav-map.tsx

```
@@ -1,90 +1,157 @@
+      : selectedStand;
+  const mapLocateRequest =
+    useMemo(() => {
+      if (!externallyLocatedEntry || !participantLocateRequest) {
+        return null;
+      }
+
+      return {
+        standId: externallyLocatedEntry.stand.id,
+        requestId: participantLocateRequest.requestId,
+      };
+    }, [externallyLocatedEntry, participantLocateRequest]) ?? locateRequest;
```

ESLint

[error] Compilation Skipped: Existing memoization could not be preserved

React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. This value was memoized in source but not in compilation output.

/home/jailuser/git/app/components/maps/festival-nav/festival-nav-map.tsx:135:5
  133 |       : selectedStand;
  134 |   const mapLocateRequest =
> 135 |     useMemo(() => {
      |     ^^^^^^^^^^^^^^^
> 136 |       if (!externallyLocatedEntry || !participantLocateRequest) {
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 137 |         return null;
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 138 |       }
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 139 |
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 140 |       return {
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 141 |         standId: externallyLocatedEntry.stand.id,
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 142 |         requestId: participantLocateRequest.requestId,
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 143 |       };
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 144 |     }, [externallyLocatedEntry, participantLocateRequest]) ?? locateRequest;
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not preserve existing memoization
  145 |
  146 |   const setActiveSector = useCallback(
  147 |     (sectorIndex: number) => {

(react-hooks/preserve-manual-memoization)